cmake/sysbuild/zip.cmake

```diff
@@ -51,19 +54,19 @@ function(dfu_app_zip_package)
   if(SB_CONFIG_DFU_ZIP_NET)
     # Network core
     get_property(image_name GLOBAL PROPERTY DOMAIN_APP_CPUNET)
+    set(net_update_name "${image_name}.bin")
     sysbuild_get(net_core_board IMAGE ${image_name} VAR BOARD CACHE)
     set(generate_script_app_params
         ${generate_script_app_params}
-        "${image_name}.binimage_index=1"
-        "${image_name}.binslot_index_primary=3"
-        "${image_name}.binslot_index_secondary=4"
-        "${image_name}.binload_address=$<TARGET_PROPERTY:partition_manager,CPUNET_PM_APP_ADDRESS>"
-        "${image_name}.binboard=${net_core_board}"
-#        "${image_name}.binversion=${net_core_version}"
```

Comment:
Hi @nordicjm,
Do you know how can we properly bring back the `"${image_name}.binversion=${net_core_version}"` property?

Reply:
`sysbuild_get(net_update_version IMAGE ${net_update_name} VAR CONFIG_FW_INFO_FIRMWARE_VERSION KCONFIG)
"${net_update_name}version=${net_update_version}"
`

Reply:
Thanks